addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/enums/SilvercrestWifiSocketVendor.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.silvercrestwifisocket.internal.enums;
+
+/**
+ * This enum represents the available Wifi Socket vendors.
+ *
+ * @author Christian Heimerl - Initial contribution
+ *
+ */
+public enum SilvercrestWifiSocketVendor {
+    Lidl_Silvercrest("C1", "7150"),

```

Comment:
Enum values should be all caps

Reply:
Why not store them as byte(s) i would assume they arrive as bytes.

Reply:
Will make it CAPS...
Basically you're right with the bytes. The message is constructed as a String, already in the versions before (e.g. in RegEx). I tried to change not too much with this concept. I would keep the String format.

Reply:
Agree on that